chemview/viewer.py

```diff
@@ -54,6 +54,31 @@ def update(self=self, points=points):
         self.update_callbacks.append(update)
         self.autozoom(self.coordinates)
         
+    def labels(self, atom_labels=None, colors=None):
+        '''Display atomic labels for the system'''
+        if not atom_labels:
+            if len(self.topology.get('atom_types'))==len(self.coordinates):
+                atom_labels=[self.topology['atom_types'][i]+str(i+1) for i in range(len(self.coordinates))]
+            else:
+                atom_labels=[str(i+1) for i in range(len(self.coordinates))]
+        if not colors:
+            colors=[{"r":255,"g":255,"b":0,"a":1} for a in self.coordinates]
+            
+        labels = self.add_representation('labels', {'coordinates': self.coordinates,
+                                           'labels': atom_labels,
+                                           'colors': colors})
+        def update(self=self, labels=labels):
+            self.update_representation(labels, {'coordinates': self.coordinates})
+        
+        self.update_callbacks.append(update)
+        self.autozoom(self.coordinates)
+        
+    def remove_labels(self):
```

Comment:
We can keep it for now but in future it might change into something more generic, so that it can be used for all representation types.